torch/package/importer.py

```diff
@@ -448,3 +459,48 @@ def patched_getfile(object):
             return _package_imported_modules[object.__module__].__file__
     return _orig_getfile(object)
 inspect.getfile = patched_getfile
+
+
+class _PackageResourceReader:
+    """Private class used to support PackageImporter.get_resource_reader().
+
+    Confirms to the importlib.abc.ResourceReader interface. Allowed to access
+    the innards of PackageImporter.
+    """
+    def __init__(self, importer, fullname):
+        self.importer = importer
+        self.fullname = fullname
+
+    def open_resource(self, resource):
+        from io import BytesIO
+        return BytesIO(self.importer.load_binary(self.fullname, resource))
```

Comment:
I think possibly we should make the `importer` API look more like the resource_reader API as well (so, `read_binary`, `open_binary` instead of `load_binary`). But for now, I just wrap it in a bytestream to conform with the interface